model/src/main/java/de/danoeh/antennapod/model/feed/FeedItem.java

```diff
@@ -42,6 +46,8 @@ public class FeedItem extends FeedComponent implements Serializable {
     private transient Feed feed;
     private long feedId;
     private String podcastIndexChapterUrl;
+    private String podcastIndexTranscriptText;
```

Comment:
Please keep the text out of the class for now, until we have decided how to actually store it.